src/Laser.cpp

```diff
@@ -11,12 +11,12 @@ Laser::Laser(int pin) : Component(pin, "LASER"){
 Laser::Laser(){
 }
 
-void Laser::engage(){
+void Laser::on(){
```

Comment:
Haha one of the points the book made was to not be cute. For example, someone may put a funny inside joke method signature inside the code not expecting it to be a source of confusion somewhere down the road. That section of the book made me specifically think about this.